src/main/java/com/connexta/store/controllers/StoreController.java

```diff
@@ -49,7 +47,7 @@ public StoreController(@NotNull final StoreService storeService) {
    */
   @Override
   public ResponseEntity<Void> storeProduct(
-      @NotBlank final String acceptVersion, @Valid @NotNull final MultipartFile file) {
+      final String acceptVersion, @Valid final MultipartFile file) {
```

Comment:
The @NotNull and @NotBlank annotations caused an org.apache.http.client.ClientProtocolException and had to be removed.